### PR TITLE
Add return types to bundles

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 6.0
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| Tickets       | #40154
| License       | MIT
| Doc PR        | -

Extracted from #42496

Bundles should be the safest to patch since their classes should be more rarely extended.

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
:green_apple:

**Review by derrabus — APPROVED** [review]

**Comment by wouterj** [comment]
Similar to #42513, if these changes are automated, I would appreciate if we can block this merge until #41613 is merged (to avoid merge conflicts in that big PR). Maybe it's good to move the SecurityBundle in the other PR, to unlock the other bundles.

**Comment by nicolas-grekas** [comment]
SecurityBundle now moved to #42513

**Pull request merged**
